.travis.yml

```diff
@@ -35,6 +35,7 @@ jobs:
   allow_failures:
     - env: CMD=pylint
     - env: CMD=bandit_site_packages
+    - env: CMD=doc8
```

Comment:
Drop this, rebase and resolve the conflicts if any.

Reply:
Done.